### DISCUSSION TITLE
Issue creating assignment - "Repository access issue"

### DISCUSSION THREAD

**Discussion by firasm** [community/community#123002] [GitHub Education] [answered]
### Hi everyone,

I'm having some problems with GitHub Classroom, specifically the "group" assignment. I've used the group assignment feature in the past and it worked fine before but currently I get an error when I try accepting the assignment and creating a group from my "test account".

<img width="763" alt="Screenshot 2024-05-08 at 9 55 53 AM" src="https://github.com/community/community/assets/2507459/b435d49b-8921-4f59-a2b9-748844b22b76">

For testing, here's the accept link: https://classroom.github.com/a/Z7Wrhi2x 

It seems the problem is that the user is never added to the team...the repo is created fine, the team is created fine, but the user is never added.

My course starts on Saturday so this is somewhat urgent - I'll try troubleshooting by opting out of the fork system for generating repos.

**Comment by firasm** [comment]
This seems to be related to #121119 and #122130

**Comment by firasm** [comment]
Update: opt-ing out of the "forks" system for provisioning repos also doesn't work (same error)... new assignment here for testing: https://classroom.github.com/a/MohHZSau

**Comment by fzle** [comment]
Hey @firasm so the above "Repository Access Issue" only appears when you try to accept a group assignment invitation with a test user? Can you please provide the username or profile link for the test user account that is currently having an issue?

**Reply by firasm** [reply, marked as answer]
Hello - the user account for my test student account is demofiras: https://github.com/demofiras

I troubleshooted it a bit more and I think the reason I had issues was because that user wasn't explicitly added to the organization. It seems this is a change - in the past, either students were automatically added to the org when they clicked the link, or it wasn't required that they're in the org. 

Would be great if I could get wonder confirmation on this, so I can adjust my workflow instructions. Ideally the error message would be a bit clearer on what the problem was and/or automatically Enrol students in the org.

**Reply by Pandeycoder** [reply]
Hi @Pandeycoder, does the 'Repository Access Issue' occur exclusively when attempting to accept a group assignment invitation using a test account? Could you please share the username or profile link of the test account experiencing this problem?
